src/contrast.js

```diff
@@ -1,11 +1,14 @@
 import parse from "./parse";
 import relativeLuminance from "./relativeLuminance";
+import { darken, lighten } from "./lightness";
 
 const contrast = (color, dark = parse("#000"), light = parse("#FFF")) => {
   const baseColor = parse(color);
   const darkColor = typeof dark === "string" ? parse(dark) : dark;
   const lightColor = typeof light === "string" ? parse(light) : light;
 
+  console.log(baseColor.luma < 128);
```

Comment:
Also, this was a quick fix in order to ship another PR of ours that relies on `polychrome`